portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro

```diff
@@ -54,7 +54,7 @@ definition {
 			var api = "asset-libraries/${assetLibraryId}/structured-content-folders";
 		}
 		else {
-			var api = "sites/${siteId}/structured-contents";
```

Comment:
is this endpoint not existing anymore? 🤔

Reply:
This is a typo, it should be with -folders

Reply:
This is a typo, it should be with -folders